Write Entry API guide about price info (#323)
cultuurnet/apidocs: projects/uitdatabank/docs/entry-api/shared/price-info.md

|---------------|--------|------------------------------------------------------------------------------------------------------------------------------------------------|------------------|
| category      | string | The first price info object has category ‘base’, all the following objects have category ‘tariff’ (differentiated price for specific segments) | base             |
| name          | array  | Category ‘base’ should always have ‘Basistarief’ for name, discounts for specific segments can be named with category ‘tariff’                 | “nl”: “Senioren” |
| price         | number | Pricevalue is a floating-point number, use 0.0 for free events/places                                                                          | 10.0             |
| priceCurrency | string | 3-characters currency indicator                                                                                                                | EUR              |

```json json_schema
{
  "type": "object",
  "properties": {
    "priceInfo": {
       "$ref": "./../../../models/common-priceInfo.json"
     }
  }
}
```

### What is the category base?

> The 'base' category the primary pricing baseline for events or places. This price will later be used for calculating discounts (eg. Uitpas tariff).

Reviewer: 2 remartks here:
* Always keep the branding in mind, it's UiTPAS 😉
* Since not all events have an UiTPAS-discount I would rephrase this.
```suggestion
The 'base' category the primary pricing baseline for events or places. If applicable, this price is also used to calculate the UiTPAS discount.```

Author: Should be fine now, but I kept the casing for this part, because the casing in the code is also just "Uitpas" and not "UiTPAS". It would be confusing to use the wrong casing here.
